Sources/Commands/SwiftPackageTool.swift

```diff
@@ -290,7 +301,13 @@ public class SwiftPackageTool: SwiftTool<PackageToolOptions> {
         parser.add(subparser: PackageMode.clean.rawValue, overview: "Delete build artifacts")
         parser.add(subparser: PackageMode.fetch.rawValue, overview: "")
         parser.add(subparser: PackageMode.reset.rawValue, overview: "Reset the complete cache/build directory")
-        parser.add(subparser: PackageMode.update.rawValue, overview: "Update package dependencies")
+
+        let updateParser = parser.add(subparser: PackageMode.update.rawValue, overview: "Update package dependencies")
+        binder.bind(
+            option: updateParser.add(
+                option: "--dry-run", kind: Bool.self,
```

Comment:
We should also add a shorthand alias "-n".

Reply:
done

Reply:
What is the reasoning behind adding an option to `swift package update` vs. a new command? `swift package list-outdated`?

Cocoapods and NPM both have a command to the same effect.